### PR TITLE
[Cyberarkpas] Update cyberarkpas to ECS 1.10.0

### PR DESCRIPTION
**Pull request by P1llus**
## What does this PR do?

Updates package ECS version to 1.10.
Sync module changes to packages if any.
Adds Preserve Raw event functionality if not already exists.
Adds pipeline tests if missing.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Related issues

- Relates #994

### CODE REVIEW

File: packages/cyberarkpas/data_stream/audit/_dev/test/pipeline/test-124-rename-file.log-expected.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
...soTimestamp\u003e\n    \u003cHostname\u003eVAULT\u003c/Hostname\u003e\n    \u003cVendor\u003eCyber-Ark\u003c/Vendor\u003e\n    \u003cProduct\u003eVault\u003c/Product\u003e\n    \u003cVersion\u003e11.7.0000\u003c/Version\u003e\n    \u003cMessageID\u003e124\u003c/MessageID\u003e\n    \u003cDesc\u003eRename File\u003c/Desc\u003e\n    \u003cSeverity\u003eInfo\u003c/Severity\u003e\n    \u003cIssuer\u003eAdministrator\u003c/Issuer\u003e\n    \u003cAction\u003eRename File\u003c/Action\u003e\n    \u003cSourceUser\u003e\u003c/SourceUser\u003e\n    \u003cTargetUser\u003e\u003c/TargetUser\u003e\n    \u003cSafe\u003ePSM\u003c/Safe\u003e\n    \u003cFile\u003eRoot\\Operating System-UnixSSH-34.123.103.115-PSMConnect\u003c/File\u003e\n    \u003cStation\u003e127.0.0.1\u003c/Station\u003e\n    \u003cLocation\u003e\u003c/Location\u003e\n    \u003cCategory\u003e\u003c/Category\u003e\n    \u003cRequestId\u003e\u003c/RequestId\u003e\n    \u003cReason\u003e\u003c/Reason\u003e\n    \u003cExtraDetails\u003e\u003c/ExtraDetails\u003e\n    \u003cMessage\u003eRename File\u003c/Message\u003e\n    \u003cGatewayStation\u003e10.0.1.20\u003c/GatewayStation\u003e\n  \u003c/audit_record\u003e\n\n\u003c/syslog\u003e",
```
Comment (andrewkroh) [inline packages/cyberarkpas/data_stream/audit/_dev/test/pipeline/test-124-rename-file.log-expected.json:47]: I wonder if we can drop this field given that we have the option of `event.original`?

**Review by andrewkroh — APPROVED** [review]
LGTM

File: packages/cyberarkpas/data_stream/audit/_dev/test/pipeline/test-124-rename-file.log-expected.json (hunk 1 above)
Comment (P1llus) [reply packages/cyberarkpas/data_stream/audit/_dev/test/pipeline/test-124-rename-file.log-expected.json:47]: Would need to check with @adriansr as he was the one who added it, might be a reason around it?

Comment (adriansr) [reply packages/cyberarkpas/data_stream/audit/_dev/test/pipeline/test-124-rename-file.log-expected.json:47]: oh, this field isn't there by default.

It's added by the XSLT for debugging purposes if necessary, to keep the original XML before the XSLT converts it to JSON.

It's added if you toggle this flag to 1:
https://github.com/elastic/beats/blob/d97155072907d26eaa27707c038161767245f6ff/x-pack/filebeat/module/cyberarkpas/_meta/assets/elastic-json-v1.0.xsl#L11

Comment (P1llus) [reply packages/cyberarkpas/data_stream/audit/_dev/test/pipeline/test-124-rename-file.log-expected.json:47]: I think its safe to have it in the test data then, or I can remove it manually if needed?

Comment (P1llus) [reply packages/cyberarkpas/data_stream/audit/_dev/test/pipeline/test-124-rename-file.log-expected.json:47]: Will leave this in the testdata for now, the field itself won't be visible for end-users, but its good to have as a reference here.

**Pull request merged**